runner/cf-driver-go/cg/cg_integration_test.go

```diff
@@ -0,0 +1,76 @@
+//go:build integration
+
+package cg_integration
+
+import (
+	"bufio"
+	"encoding/json"
+	"os"
+	"testing"
+
+	"github.com/GSA-TTS/gitlab-runner-cloudgov/runner/cf-driver/cg"
+	"github.com/google/go-cmp/cmp"
+)
+
+func Test_CFAdapter_GetApps(t *testing.T) {
+	var u, p, want string
+	var l int
+
+	f, err := os.Open("./testdata/.cg_creds")
+	if err != nil {
+		t.Errorf("Error opening testdata file = %v", err)
+	}
+	defer f.Close()
+
+	scanner := bufio.NewScanner(f)
+
+scanning:
```

Comment:
[question] the indentation of this looks weird, is it?

Reply:
I would agree, but that's the official style. It is fairly logical, because usually people don't use labels for loops and the idea, I think, is to make them stand out.